php/class-plugin.php

```diff
@@ -13,38 +13,30 @@
  * Class Plugin
  */
 class Plugin extends Plugin_Abstract {
-	/**
-	 * WP Admin resources.
-	 *
-	 * @var Admin\Admin
-	 */
-	public $admin;
 
 	/**
-	 * Block Post Type.
+	 * Utility methods.
 	 *
-	 * @var Post_Types\Block_Post
+	 * @var Util
 	 */
-	public $block_post;
+	protected $util;
 
 	/**
-	 * Initiate the loading of new blocks.
+	 * WP Admin resources.
 	 *
-	 * @var Blocks\Loader
+	 * @var Admin\Admin
 	 */
-	public $loader;
+	public $admin;
 
 	/**
 	 * Execute this as early as possible.
 	 */
 	public function init() {
-		$this->set_util();
-
-		$this->block_post = new Post_Types\Block_Post();
-		$this->register_component( $this->block_post );
+		$this->util = new Util();
+		$this->register_component( $this->util );
 
-		$this->loader = new Blocks\Loader();
-		$this->register_component( $this->loader );
+		$this->register_component( new Post_Types\Block_Post() );
```

Comment:
Nice work removing the `block_post` and `loader` properties.

Reply:
Thanks 😄